lib/hodor/util/yml_tools.rb

```diff
@@ -1,36 +1,17 @@
 require 'yaml'
-require 'erb'
+require_relative 'erb_tools'
 
-module Hodor
-  class Configuration
+module Hodor::Util
+  module YmlTools
+    include ErbTools
 
-    def env
-      Environment.instance
-    end
-
-    def target
-      env.settings[:target]
-    end
-
-    def logger
-      env.logger
-    end
-
-    def initialize(yml_file)
-      @yml_file = yml_file
-      @kvp = {}
-    end
-
-    def load
-
-      @loaded = true
-
-      yml_expand(@target_cluster, [@clusters])
+    def yml_load(filename) #, suppress_erb=false)
+      YAML.load(erb_load(filename, false)) # suppress_erb))
     end
 
     def yml_expand(val, parents)
```

Comment:
Another reminder: YmlTools and yml_expand have become a misnomer now, with your generalization of ConfigSet Sources, since these messages (for the most part) only manipulate hashes, and are agnostic to YAML.